content/en/account_management/org_settings/cross_org_visibility.md

```diff
@@ -1,12 +1,11 @@
 ---
 title: Cross-Organization Visibility
 kind: documentation
-private: true
 is_beta: true
 ---
 
 {{< callout url="#" header="false" btn_hidden="true">}}
-  Cross-organization visibility is in <strong>private beta</strong> for customers with Enterprise plans. If you're interested in the feature, reach out to your Technical Account Manager or Customer Success Manager.
+  Note: Cross-organization visibility is in <strong>public beta</strong>. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```

Comment:
```suggestion
  Note: Cross-organization visibility is in beta. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```
We don't usually specify "public" in these.

Reply:
> We don't usually specify "public" in these.
Except for all the places where we do: https://www.google.com/search?q=site%3Adocs.datadoghq.com+%22public+beta%22

Accepting this one